src/init.js

```diff
@@ -74,7 +82,7 @@ export default ({ clientSideId, dispatch, flags, user, subscribe, options }) =>
   window.ldClient = ldClientInitialize(clientSideId, user, options);
   window.ldClient.on('ready', () => {
     const flagsSanitised = flags || ldClient.allFlags();
-    setFlags(flagsSanitised, dispatch);
+    setFlags(flagsSanitised, dispatch, camelCased);
```

Comment:
I couldn't find an easy way to test this without exporting the `setFlags` function and then importing it into the test file so it can be used in the mock callback. I avoided this because `setFlags` is private and shouldn't be exposed as a public method. One alternative is this could live in another file that `init.js` and `init.test.js` both import. Thoughts?

Reply:
I think exporting `setFlags` to test it is fine.